nominal/core/filetype.py

```diff
@@ -87,6 +115,16 @@ class FileTypes:
     MCAP: FileType = FileType(".mcap", "application/octet-stream")
     # https://issues.apache.org/jira/browse/PARQUET-1889
     PARQUET: FileType = FileType(".parquet", "application/vnd.apache.parquet")
+    PARQUET_GZ: FileType = FileType(".parquet.gz", "application/octet-stream")
+    PARQUET_TAR_GZ: FileType = FileType(".parquet.tar.gz", "application/x-tar")
+    PARQUET_TAR: FileType = FileType(".parquet.tar", "application/x-tar")
+    PARQUET_ZIP: FileType = FileType(".parquet.zip", "application/zip")
     TS: FileType = FileType(".ts", "video/mp2t")
     JOURNAL_JSONL: FileType = FileType(".jsonl", "application/jsonl")
     JOURNAL_JSONL_GZ: FileType = FileType(".jsonl.gz", "application/jsonl")
+
+    CSV_TYPES = (CSV, CSV_GZ)
```

Comment:
maybe we should `_` these or move them to a diff namespace since `FileTypes` is public?